src/sections/speakers.astro

```diff
@@ -15,13 +15,13 @@ const support = await getCollection("support");
 </div>
 
 <ul role="list" class="person-list">
-  {
+  <!-- {
```

Comment:
I think this needs to be reverted right?

Reply:
Fixed, thanks!